src/components/DelegatesTable/DelegateVoteCheckbox.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <v-icon
+    v-if="voted"
+    :class="classes.icon"
+    icon="mdi-thumb-up"
+    size="large"
+    @click="handleDownVote"
+  />
+  <v-icon
+    v-else
+    :class="classes.icon"
+    icon="mdi-thumb-up-outline"
+    size="large"
+    @click="handleUpVote"
+  />
+</template>
+
+<script>
+import { computed } from 'vue'
+import { useStore } from 'vuex'
+
+export default {
+  props: {
+    delegate: {
+      type: Object,
+      required: true
+    }
+  },
+  setup (props) {
+    const { delegate } = props
+    const store = useStore()
```

Comment:
This is a subject for discussion. I would keep the old eslint configuration for now, otherwise it will introduce more diffs and the PR will be even harder to review.

Reply:
The majority of the changes are related to HTML and CSS, with minimal logic code. Given that there is a proper ESLint configuration in place, I believe a full code review may not be necessary.

Reply:
@martiliones I linted all the SCSS files. Check again